setup.py

```diff
@@ -40,7 +40,7 @@
     "chex",
     # TODO Unpin JAX, once new rlax and acme version is released.
     # To fix cannot import name 'tree_multimap' from 'jax.tree_util'.
-    "jax<=0.3.15",
+    "jax==0.3.17",
```

Comment:
Should the setup.py file use `<=` instead of `==` to reduce requirement version issues for local installations?

Reply:
Yeah you are right I will bring it back to <=0.3.15